mobile/lib/domain/services/sync_stream.service.dart

```diff
@@ -139,14 +139,18 @@ class SyncStreamService {
         return _syncStreamRepository.updateAlbumUsersV1(data.cast(), debugLabel: 'backfill');
       case SyncEntityType.albumUserDeleteV1:
         return _syncStreamRepository.deleteAlbumUsersV1(data.cast());
-      case SyncEntityType.albumAssetV1:
```

Comment:
Should we move this down to the section of `TO BE REMOVED` so that the user can keep using the `v1.137.x` if they haven't updated yet

Reply:
I don't know how much I want to maintain backwards compatibility for the beta timeline, doing this means we need to keep the types in the server there so they are generated too. I would just put a note that says albums on beta timeline need you to update to continue working